code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -79,6 +79,17 @@
 				else
 					C.deconstruct()
 					visible_message("<span class='warning'>[src] chews through the [C].</span>")
+	for(var/obj/item/reagent_containers/food/snacks/cheesewedge/cheese in range(1, src))
+		if(prob(10))
+			var/cap = CONFIG_GET(number/ratcap)
+			if(LAZYLEN(SSmobs.cheeserats) > cap)
+				visible_message("<span class='warning'>[src] carefully eats the cheese, hiding from the [cap] mice on the station!</span>")
```

Comment:
> hiding from the 64 mice on the station

Hiding what? The cheese? The mouse doesnt go anywhere right?

Reply:
The mouse is hiding the cheese from the other mice on station, so that there's less competition.